iocore/net/UnixNetAccept.cc

```diff
@@ -330,10 +330,10 @@ NetAccept::acceptEvent(int event, void *ep)
     if ((res = accept_fn(this, e, false)) < 0) {
       NET_DECREMENT_DYN_STAT(net_accepts_currently_open_stat);
       /* INKqa11179 */
-      Warning("Accept on port %d failed with error no %d", ats_ip_port_host_order(&server.addr), res);
+      Warning("Accept on port %d failed with error no %d", ats_ip_port_host_order(&server.accept_addr), res);
```

Comment:
`accept_addr` is used for configure, and it will be passed to `server.addr` in `[Server::listen](https://github.com/apache/trafficserver/blob/master/iocore/net/Connection.cc#L283)` if the `accept_addr` is valid.

I think the original logic is correct!

Reply:
you are right, the ```Server::listen()``` makes them no different.

I'm just reference the code from ```NetAccept::listen()```.

```
200 int
201 NetAccept::do_listen(bool non_blocking)
202 {
203   int res = 0;
204 
205   if (server.fd != NO_FD) {
206     if ((res = server.setup_fd_for_listen(non_blocking, opt))) {
207       Warning("unable to listen on main accept port %d: errno = %d, %s", ntohs(server.accept_addr.port()), errno, strerror(errno));
208       goto Lretry;
209     }
210   } else {
211   Lretry:
212     if ((res = server.listen(non_blocking, opt))) {
213       Warning("unable to listen on port %d: %d %d, %s", ntohs(server.accept_addr.port()), res, errno, strerror(errno));
214     }
215   }
```
Thanks.